src/store/TableDataStore.js

```diff
@@ -18,6 +18,7 @@ export class TableDataStore {
     this.pageObj = {};
     this.selected = [];
     this.multiColumnSearch = false;
+    this.strictSearch = true;
```

Comment:
According to that reasoning we can also remove the `this.multiColumnSearch = false` line above.
I just wanted to be consistent 😉